code/modules/projectiles/gun_attachables.dm

```diff
@@ -1275,7 +1275,8 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 	flags_attach_features = ATTACH_REMOVABLE|ATTACH_ACTIVATION|ATTACH_RELOADABLE|ATTACH_WEAPON
 	///list of grenade types loaded in the UGL
 	var/list/loaded_grenades = list()
-	attachment_firing_delay = 21
+	attachment_firing_delay = 10
+	COOLDOWN_DECLARE(last_fired)
```

Comment:
declaring new vars comes under redefining new vars, move attachment_firing_delay

Reply:
This is an exact copy + paste of underbarrel flamer code. If you want me to change this then I'll change under barrel flamer code as well.

Reply:
Should be fixed now.